--- a/setup.py	2019-06-21 21:56:59.543000000 -0300
+++ b/setup.py	2019-06-21 21:57:13.760000000 -0300
@@ -129,21 +129,21 @@
 setup_args['cmdclass'] = {
     'build_py': css_js_prerelease(
             check_package_data_first(build_py)),
     'sdist' : css_js_prerelease(sdist, strict=True),
     'develop': css_js_prerelease(develop),
     'css' : CompileCSS,
     'backendtranslations': CompileBackendTranslation,
     'js' : CompileJS,
     'jsdeps' : Bower,
     'jsversion' : JavascriptVersion,
-    'bdist_egg': bdist_egg if 'bdist_egg' in sys.argv else bdist_egg_disabled,
+#    'bdist_egg': bdist_egg if 'bdist_egg' in sys.argv else bdist_egg_disabled,
 }
 
 try:
     from wheel.bdist_wheel import bdist_wheel
 except ImportError:
     pass
 else:
     setup_args['cmdclass']['bdist_wheel'] = css_js_prerelease(bdist_wheel)
 
 # Run setup --------------------
